### ISSUE TITLE
Accessibility: Mini cart remove button should be a button not a link

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#56606]
In block-based themes, the remove button is a button. This is a good example:

<img width="1578" alt="Image" src="https://github.com/user-attachments/assets/0617dcfd-061a-4b2b-b157-ea7ae3b0ba30" />

## Problem
In themes like Astra that call WC_Widget_Cart, the remove product button is incorrectly coded as a link.

https://github.com/woocommerce/woocommerce/blob/d1e8f3832d2a12141ac0b1f1667460ded8905f0a/plugins/woocommerce/templates/cart/mini-cart.php#L50

**Comment by amberhinds** [comment]
@jeffpaul @vikrampm1 can you work on this one? It was just discovered while testing in a non-block based theme.

**Comment by vikrampm1** [comment]
@amberhinds thanks for flagging, Gabriel should be able to look into it next week.

**Comment by Manussakis** [comment]
Hello @amberhinds 
Thanks for opening this issue. I struggle to display the mini-cart on the header using the Astra theme.
Could you let me know the steps to reproduce it on Astra?

Thanks!

**Comment by amberhinds** [comment]
Here's how to insert the WooCommerce Cart in the header in Astra.

1. Go to Appearance > Customize
2. Click Header Builder
3. In the header builder section that opens at the bottom of the main content in the Customizer, hover over a column to see a + icon appear.
4. Click on the column and an "Insert Elements" modal appears.
5. In the insert elements modal, click on the cart. This will insert the cart in the header.
6. Publish changes.

<img width="1806" alt="Image" src="https://github.com/user-attachments/assets/6307fb91-2dcc-4e7d-9d97-d1c191ec1815" />

**Comment by Manussakis** [comment]
Thank you, @amberhinds !
